nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -62,22 +63,26 @@ define([
         /*
             node: container node to build the widget into
 
-            This is more or less copied over from the way that the appCell handles building the widget. This model assumes one instance of the params widget per cell, so may need some adjustment as we make the bulk import work with multiple data types
+            This is more or less copied over from the way that the appCell handles building the widget.
+            This model assumes one instance of the params widget per cell, so may need some adjustment as
+            we make the bulk import work with multiple data types
         */
         function buildParamsWidget(node) {
             const paramBus = runtime.bus().makeChannelBus({ description: 'Parent comm bus for input widget' });
+            // This is the key in the model that maps to the list of params for the current app.
+            const paramKey = `params.${fileType}`;
```

Comment:
I'm not sure if there's a better way to do this. Right now in the data model, params are keyed under each filetype. The fileType is the main key for all the other data - it maps to the app spec ids, the files, and to which params are active.

Reply:
i think this approach makes sense, especially as we agreed that for now we will only have one set of params per file type